code/src/components/happyForm.css

```diff
@@ -0,0 +1,41 @@
+
+
+.happy-form {
+    border: 1px solid black;
+    background: #F2F0F0;
+    width: 350px;
+    display: inline-block;
+    padding: 20px;
+    box-shadow: 5px 10px;
+    font-family: 'Inconsolata', monospace;
+}
+
+textarea {
+    width: 90%;
+    
+}
+
+.form-footer {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 10px;
+    color:grey;
+}
+
+.form-footer button {
+    height: 35px;
+    width: 200px;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 25px;
+    border: none;
+    background-color:#F4ACAC;
+    color: black;
+    font-family: 'Inconsolata', monospace;
+}
+
+.form-footer button:hover {
```

Comment:
Nice hovering over the button! 😍 Would love to see the hovering over the heart buttons aswell - like the end result picture in the projects brief ! 😁